crates/re_viewer/src/ui/welcome_screen/example_page.rs

```diff
@@ -0,0 +1,391 @@
+use re_ui::ReUi;
+use re_viewer_context::SystemCommandSender;
+use std::collections::HashMap;
+
+// TODO(emilk/egui#3291): replace this by loading image from the web
+static THUMBNAIL_CACHE: once_cell::sync::Lazy<HashMap<&'static str, re_ui::icons::Icon>> =
+    once_cell::sync::Lazy::new(|| {
+        [
+            re_ui::icons::Icon::new(
+                "efb301d64eef6f25e8f6ae29294bd003c0cda3a7_detect_and_track_objects_480w.png",
+                include_bytes!(
+                    "../../../data/example_thumbnails/efb301d64eef6f25e8f6ae29294bd003c0cda3a7\
+                    _detect_and_track_objects_480w.png"
+                ),
+            ),
+            re_ui::icons::Icon::new(
+                "8b90a80c72b27fad289806b7e5dff0c9ac97e87c_arkit_scenes_480w.png",
+                include_bytes!(
+                    "../../../data/example_thumbnails/8b90a80c72b27fad289806b7e5dff0c9ac97e87c\
+                    _arkit_scenes_480w.png"
+                ),
+            ),
+            re_ui::icons::Icon::new(
+                "033edff752f86bcdc9a81f7877e0b4411ff4e6c5_structure_from_motion_480w.png",
+                include_bytes!(
+                    "../../../data/example_thumbnails/033edff752f86bcdc9a81f7877e0b4411ff4e6c5\
+                    _structure_from_motion_480w.png"
+                ),
+            ),
+            re_ui::icons::Icon::new(
+                "277b9c72da1d0d0ae9d221f7552dede9c4d5b2fa_human_pose_tracking_480w.png",
+                include_bytes!(
+                    "../../../data/example_thumbnails/277b9c72da1d0d0ae9d221f7552dede9c4d5b2fa\
+                    _human_pose_tracking_480w.png"
+                ),
+            ),
+            re_ui::icons::Icon::new(
+                "b8b25dd01e892e6daf5177e6fc05ff5feb19ee8d_dicom_mri_480w.png",
+                include_bytes!(
+                    "../../../data/example_thumbnails/b8b25dd01e892e6daf5177e6fc05ff5feb19ee8d\
+                    _dicom_mri_480w.png"
+                ),
+            ),
+            re_ui::icons::Icon::new(
+                "ca0c72df93d70c79b0e640fb4b7c33cdc0bfe5f4_plots_480w.png",
+                include_bytes!(
+                    "../../../data/example_thumbnails/ca0c72df93d70c79b0e640fb4b7c33cdc0bfe5f4\
+                    _plots_480w.png"
+                ),
+            ),
+            re_ui::icons::Icon::new(
+                "ea7a9ab2f716bd37d1bbc1eabf3f55e4f526660e_helix_480w.png",
+                include_bytes!(
+                    "../../../data/example_thumbnails/ea7a9ab2f716bd37d1bbc1eabf3f55e4f526660e\
+                    _helix_480w.png"
+                ),
+            ),
+        ]
+        .into_iter()
+        .map(|icon| (icon.id, icon))
+        .collect()
+    });
+
+#[derive(Debug, serde::Deserialize)]
+struct ExampleThumbnail {
+    url: String,
+    width: u32,
+    height: u32,
+}
+
+#[derive(Debug, serde::Deserialize)]
+#[allow(unused)]
+struct ExampleDesc {
+    name: String,
+    title: String,
```

Comment:
what's the difference?

Reply:
Name is a snake-case identifier of sort, and title is the actual user-facing string. I need to stick to the JSON conventions here (can updated that in another PR), but I'll add some clarifying comments.